en/orm/behaviors/tree.rst

```diff
@@ -95,6 +95,11 @@ in a hierarchy, you can stack the 'threaded' finder::
         echo "{$child->name} has " . count($child->children) . " direct children";
     }
 
+While, if you’re using custom ``parent_id`` you need to pass it in the 'threaded' finder option (i.e. ``parentField``) . 
+
+.. note::
+    For more information on 'threaded' finder options see :ref:`Finding Threaded Data logic <finding-threaded-data>`
```

Comment:
The `finding-threaded-data` ref doesn't exist. Need to add it above header in https://github.com/cakephp/docs/blob/4.x/en/orm/retrieving-data-and-resultsets.rst

```
.. _finding-threaded-data:

Finding Threaded Data
=====================
```

Reply:
https://github.com/cakephp/docs/pull/7422#issue-1238439633